### PR TITLE
LZ-LB-add first item button

### LINKED ISSUE #7: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works
AC:

* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

### PR DESCRIPTION
**Pull request by lilyhoratio**
## Description

We rerouted the "add new list" button from AddItem component to the List component. In the List component, we conditionally rendered to show "add first item" button when the shopping list is empty. 

## Related Issue

Closes #7 

## Acceptance Criteria

- The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|   ✓ | :hammer: Refactoring       |

### CODE REVIEW

**Comment by abidishajia** [comment]
Simple and clean. Love it!

File: src/components/Welcome.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Welcome = ({ firestore }) => {
     const newToken = GetToken();
     saveToken(newToken);
     setToken(newToken);
-    push('/addItem');
+    push('/list');
```
Comment (stacietaylorcima) [inline src/components/Welcome.js:55]: Yes! I love that you're sending them through the list first and then to the addItem view from there. Sending the user through a nice flow like this is training them how to use the tool. 👏

File: src/components/List.js [hunk 2]
```diff
@@ -12,14 +22,18 @@ const List = () => {
...
-                {data.map(item => (
-                  <li key={item.id}>
-                    {item.name} - next purchase in {item.nextPurchase} days
-                  </li>
-                ))}
-              </ul>
+            <div className="List">
+              {!data.length ? (
+                <button onClick={handleClick}>Add First Item</button>
+              ) : (
```
Comment (stacietaylorcima) [inline src/components/List.js:28]: Heck yesss! So clean and simple and I LERV IT! ✨

File: src/components/List.js [hunk 3]
```diff
@@ -12,14 +22,18 @@ const List = () => {
...
           ) : (
-            <div>
-              <ul style={{ listStyleType: 'none' }}>
-                {data.map(item => (
-                  <li key={item.id}>
-                    {item.name} - next purchase in {item.nextPurchase} days
-                  </li>
-                ))}
-              </ul>
+            <div className="List">
```
Comment (stacietaylorcima) [inline src/components/List.js:25]: Not a blocker by any means, you've totally met the Acceptance Criteria, but I might **suggest** adding just a bit more detail here for the user to let them know what they're seeing. This is what one of the last cohorts put, but it's totally up to you. Kinda put yourself in the shoes of a user and think about what you'd wanna see here to guide you through using the app. 🤔

Comment (Lukabaramishvili) [reply src/components/List.js:25]: Absolutely. Thanks for this reminder.

**Pull request merged**